### ISSUE TITLE
"No date for motExpiryDate" recurs in log file for vehicles not needing an MOT

### ISSUE THREAD

**Issue by steve-hodgkiss** [jampez77/DVLA-Vehicle-Enquiry-Service#12]
Have an EV car less than three years old.

In log file this message keeps getting logged:
```
2024-02-12 14:37:22.629 WARNING (MainThread) [custom_components.dvla.calendar] No date for motExpiryDate
```
Sensor for motExpiryDate is created with a value of unknown in HA. Suggest sensor should only be created if Mot States not eq "No details held by DVLA". Or keep the sensor and suppress the log message.

**Comment by chrisandsally** [comment]
Same issue.

**Comment by jampez77** [comment]
This should be fixed in [2024.7.0](https://github.com/jampez77/DVLA-Vehicle-Enquiry-Service/releases/tag/2024.7.0)
